fixtures/coverall/tests/bindings/test_coverall.rb

```diff
@@ -98,17 +98,19 @@ def test_simple_errors
     coveralls = Coverall::Coveralls.new 'test_simple_errors'
     assert_equal coveralls.get_name, 'test_simple_errors'
 
-    assert_raise Coverall::CoverallError::TooManyHoles do
+    err = assert_raise Coverall::CoverallError::TooManyHoles do
       coveralls.maybe_throw true
     end
+    assert_equal err.message, 'The coverall has too many holes'
```

Comment:
My initial attempt at making a failing testcase here didn't fail, which lead me to discover that we're using the wrong syntax for asserting the error message in Ruby. I tested this new syntax on the existing assertion below to confirm that it works.